ui/src/submissions/index.jsx

```diff
@@ -115,6 +123,11 @@ class Submissions extends Component {
               path={`${SUBMISSIONS_SEMINAR}/new/success`}
               component={SeminarSubmissionSuccessPageContainer}
             />
+            <Route
+              exact
+              path={`${SUBMISSIONS_EXPERIMENT}/new/success`}
+              component={ExperimentSubmissionSuccessPageContainer}
+            />
```

Comment:
should be private https://github.com/inspirehep/inspirehep/blob/ba5fe0fdd9aa4daebad985376068487edccf995a/ui/src/common/PrivateRoute.jsx#L10